### ISSUE TITLE
Improve package not found error

### ISSUE THREAD

**Issue by chdoig** [conda/conda#4872]
Improve package not found error: 

Conda could not find <empty_string>:

2017-03-14 14:25:11,982 - ERROR packages._conda_process_ready:633
('PackageNotFoundError', "Package not found: Conda could not find '")

2017-03-14 14:25:12,002 - ERROR environments._environment_created:483
/Users/cdoig/anaconda/bin/python /Users/cdoig/anaconda/bin/conda create --yes --json --mkdir --name new-env-test python=3.6 --override-channels --channel https://conda.anaconda.org/r --channel https://conda.anaconda.org/anaconda-fusion: Package not found: Conda could not find '

**Comment by AjayTalati** [comment]
Oh dear ? I got the same too ,

```
(pyphi) ajay@ajay-h8-1170uk:~/RProjects/Small_phi$ conda update conda
PackageNotFoundError: Package not found: Conda could not find 'conda
```

**Comment by orodbhen** [comment]
Is there any solution to this, to get it working again at least?

**Comment by ronert** [comment]
Same here

```
$ conda install fuzzywuzzy
Fetching package metadata .........


PackageNotFoundError: Package not found: Conda could not find '
```

**Comment by kalefranz** [comment]
Fixed in (I think) 4.3.15.  Definitely fixed though.

**Comment by kalefranz** [comment]
Sorry, it was 4.3.16 and PR #4935
